release/docker/config/opensearch/opensearch-docker-entrypoint.sh

```diff
@@ -71,6 +74,25 @@ done < <(env)
 # will run in.
 export OPENSEARCH_JAVA_OPTS="-Dopensearch.cgroups.hierarchy.override=/ $OPENSEARCH_JAVA_OPTS"
 
+##Security Plugin
+SECURITY_PLUGIN="opensearch-security"
+if [ -d "$OPENSEARCH_HOME/plugins/$SECURITY_PLUGIN" ]; then
+    if [ "$DISABLE_INSTALL_DEMO_CONFIG" = "true" ]; then
+        echo "Disabling Install Demo Config for OpenSearch Security Plugin"
+    else
+        echo "Enabling Install Demo Config for OpenSearch Security Plugin"
+        bash $OPENSEARCH_HOME/plugins/$SECURITY_PLUGIN/tools/install_demo_configuration.sh -y -i -s
+    fi
+
+    if [ "$DISABLE_SECURITY_PLUGIN" = "true" ]; then
+        echo "Disabling OpenSearch Security Plugin"
+        sed -i '/plugins.security.disabled/d' $OPENSEARCH_HOME/config/opensearch.yml
+        echo "plugins.security.disabled: true" >> $OPENSEARCH_HOME/config/opensearch.yml
+    else
+        echo "Enabling OpenSearch Security Plugin"
+        sed -i '/plugins.security.disabled/d' $OPENSEARCH_HOME/config/opensearch.yml
```

Comment:
Why delete this config when the security plugin is enabled? If customer wants to change it in future (disable it later) he may not find the config in `opensearch.yml` which can be confusing

Reply:
I will change it to false then.

Reply:
Thinking back this line should not be there on the 1st place since enabling means we are using the security plugin. If we add a false then it is different from the tarball and only add up confusion.

I disagree with this and will just remove the config.

Reply:
Agreed! Do nothing is good